nbqa/__main__.py

```diff
@@ -224,6 +234,7 @@ def _get_mtimes(arg: str) -> Set[float]:
 
 def _run_command(
     command: str,
+    shell: bool,
     cmd_args: Sequence[str],
     args: Sequence[str],
 ) -> Tuple[Output, int, bool]:
```

Comment:
[boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap), let's rewrite as:
```suggestion
def _run_command(
    command: str,
    cmd_args: Sequence[str],
    args: Sequence[str],
    *,
    shell: bool,
) -> Tuple[Output, int, bool]:
```